run/test/specs/voice_calls.spec.ts

```diff
@@ -0,0 +1,211 @@
+// import { androidIt, iosIt } from "../../types/sessionIt";
+// import { newUser } from "./utils/create_account";
+// import { runOnlyOnAndroid, sleepFor } from "./utils/index";
+// import {
+//   SupportedPlatformsType,
+//   closeApp,
+//   openAppTwoDevices,
+// } from "./utils/open_app";
+
+// androidIt("Voice calls", voiceCallAndroid);
+// iosIt("Voice calls", voiceCallIos);
+
+// async function voiceCallIos(platform: SupportedPlatformsType) {
```

Comment:
to keep disabled?

Reply:
Voice calls are really unstable right now, so keeping it skipped for now